ddterm/com.github.amezin.ddterm.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Drop Down Terminal
+NoDisplay=true
+Comment=Another drop down terminal extension for GNOME Shell. With tabs. Works on Wayland natively.
+Icon=terminal
+Exec=gdbus call --session --dest org.gnome.Shell --object-path /org/gnome/Shell/Extensions/ddterm --method com.github.amezin.ddterm.Extension.Toggle
```

Comment:
`Activate` method might be more appropriate here

Reply:
From what I read in the [Freedesktop Spec](https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html), the line `DBusActivatable=true` is needed to signal that ddterm can be activated via dbus. Besides the `Activate` method, they suggest implementing an `Open` method for opening files with the application and an `ActivateAction` for desktop actions. Since the last two are not relevant for ddterm, I think we're good without them.

The `Exec` line must be kept anyway, to keep compatibility with implementations that don't handle the `DBusActivatable` line properly.

Reply:
No, I mean `Activate` method from ddterm extension's D-Bus interface. `Toggle` hides ddterm if it's visible. `Activate` always shows/brings the window to the front.

No need for D-Bus activation (it can't be correctly implemented here anyway).

Reply:
Oh, of course 🤦🏻‍♂️
Sorry, I misunderstood you.